### PR TITLE
Fixes cyborg crew monitor interaction

### LINKED ISSUE #35641: sillycons can't use crew monitoring consoles
does literally nothing on click

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
fix: After a minor firmware update crew monitoring consoles now work for cyborgs.
/:cl:

Fixes: #35641 

World.view is a string: "15x15"
getveiwsize() does some math I don't trust.

### CODE REVIEW

**Comment by francinum** [comment]
the CMC needs to have a UI redo, it's operating on some really weird not-tgui system.

**Comment by Supermichael777** [comment]
way beyond my skill

**Comment by YPOQ** [comment]
BYOND doc says range accepts strings
>You can also use a rectangular box size using a text string such as "13x11". Both arguments are optional and may be passed in any order.

File: code/game/machinery/computer/crew.dm [hunk 1]
```diff
@@ -221,7 +221,7 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		if (isAI(hclient.client.mob))
 			return TRUE
 		else if (iscyborg(hclient.client.mob))
-			return (locate(/obj/machinery/computer/crew, range(world.view, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents))
+			return (locate(/obj/machinery/computer/crew, range(7, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents))
```
Comment (vuonojenmustaturska) [inline code/game/machinery/computer/crew.dm]: use `getviewsize()`, which returns a list with X and Y view sizes. Take the max() of those sizes, and round half of it. That's your view radius.

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
ping for reopen or make new pr when you're ready to conintue

**Pull request closed without merging**